src/components/calculator.js

```diff
@@ -1,45 +1,44 @@
+/* eslint-disable import/no-unresolved */
 /* eslint-disable no-unused-vars */
 import React, { useState } from 'react';
-import '../calculator.css';
+import Screen from './Screen.js';
+import Button from './Button.js';
 import calculate from '../logic/calculate.js';
 
 const Calculator = () => {
```

Comment:
- [ ] As stated in the requirements, while you are encouraged to style the app as you like, your general layout of the pages should still match the wireframe. In your **Calculator** page, your calculator is somewhat centered. Kindly move your calculator to the right side of the page to match the wireframe provided.

  Your Implementation | Provided wireframe
  :---:|:---:
  ![tumwo-des](https://user-images.githubusercontent.com/36542986/149501936-b5f110e7-ed6c-4fbe-9962-de4cf643d072.jpg) | ![vicma-calc-temp](https://user-images.githubusercontent.com/36542986/149501942-214865ec-a49e-435f-b99f-1ed8b476774b.jpg)